crates/noirc_evaluator/src/ssa_refactor/ssa_gen/program.rs

```diff
@@ -34,10 +34,31 @@ impl Ssa {
         &self.functions[&self.main_id]
     }
 
+    /// Returns the function with the given ID
+    pub(crate) fn get_fn(&self, id: FunctionId) -> &Function {
+        self.functions.get(&id).expect("ICE: Could not find function")
+    }
+
     /// Returns the entry-point function of the program as a mutable reference
     pub(crate) fn main_mut(&mut self) -> &mut Function {
         self.functions.get_mut(&self.main_id).expect("ICE: Ssa should have a main function")
     }
+
+    /// Returns the function with the given ID as a mutable reference
+    pub(crate) fn get_fn_mut(&mut self, id: FunctionId) -> &mut Function {
+        self.functions.get_mut(&id).expect("ICE: Could not find function")
+    }
```

Comment:
this function is unused now

Reply:
Do we have misconfigured clippy? I didn't get a clippy warning for this

Reply:
Possibly, I don't think clippy normally only warns about this if the function is completely private, but maybe it is a setting somewhere.